recipes-devtools/deepstream/deepstream-5.0_5.0.0-1.bb

```diff
@@ -0,0 +1,73 @@
+DESCRIPTION = "NVIDIA Deepstream SDK"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://usr/share/doc/deepstream-5.0/copyright;md5=f635f9f375e764ce281a2070599e2457"
```

Comment:
Not a showstopper for this, but (not surprisingly) there's a bunch of open-source code included in the package as well, and additional license texts:
```
/opt/nvidia/deepstream/deepstream-5.0/LICENSE.txt
/opt/nvidia/deepstream/deepstream-5.0/doc/nvidia-tegra/LICENSE.nvds_amqp_protocol_adaptor
/opt/nvidia/deepstream/deepstream-5.0/doc/nvidia-tegra/LICENSE.iothub_client
```
It would be good to enumerate them all and add the texts to the checksum.